src/org/traccar/reports/ReportUtils.java

```diff
@@ -35,4 +39,46 @@ private ReportUtils() {
         return result;
     }
 
+    public static double calculateDistance(Position firstPosition, Position lastPosition) {
+        return calculateDistance(firstPosition, lastPosition, true);
+    }
+
+    public static double calculateDistance(Position firstPosition, Position lastPosition, boolean useOdometer) {
+        BigDecimal distance = new BigDecimal("0.0");
```

Comment:
BigDecimal can take double value. String looks weird. Same for "1000" below. It can just be int.